cli/command/formatter/stats.go

```diff
@@ -37,11 +35,80 @@ type ContainerStatsAttrs struct {
 	PidsCurrent      uint64 // Not used on Windows
 }
 
-// ContainerStats represents the containers statistics data.
+// ClearContainerStats represents the statistics data collected from a container.
+type ClearContainerStats struct {
```

Comment:
Yes, I am bad at giving names :blush:.
Is it a bad idea to move this to api/types and call it `ContainerStats`?